Package.swift

```diff
@@ -0,0 +1,10 @@
+// swift-tools-version:5.1
+import PackageDescription
+
+let package = Package(name: "StickyTabBarViewController",
+                      platforms: [.iOS(.v10)],
+                      products: [.library(name: "StickyTabBarViewController",
```

Comment:
Should we define a global private constant in this file for this string?

Reply:
Good idea, done here https://github.com/emrepun/StickyTabBarViewController/pull/25/commits/ff0dcdb13bebb48dd4b6382f67d3f189b4a4eb38 @Thurman1776-